be/src/exec/vectorized/topn_node.cpp

```diff
@@ -147,7 +147,7 @@ Status TopNNode::_consume_chunks(RuntimeState* state, ExecNode* child) {
     bool eos = false;
     _chunks_sorter->setup_runtime(runtime_profile(), "ChunksSorter");
     do {
-        RETURN_IF_ERROR(state->check_query_state("IntersectNode"));
```

Comment:
there are 3 `check_query_state` in this file, why only changes one?

Reply:
The place where mem check is currently added:
* PlanFragmentExecutor::get_next()
* Join, CrossJoin, TopN open()-->get_next_child_chunk()